setup.cfg

```diff
@@ -1,8 +1,18 @@
+[tool:pytest]
+testpaths = resample
+addopts = --doctest-modules
+
 [mypy]
 ignore_missing_imports = True
+allow_redefinition = True
 
 [flake8]
 max_line_length = 88
+ignore = E203
+exclude = setup.py
+
+[pylint]
+rcfile = .pylintrc
```

Comment:
I'm actually not familiar with pylint. What is this adding on top of black / flake8?

Reply:
pylint also complains about stylistic issues, one-letter variables, for example.
https://www.reddit.com/r/Python/comments/82hgzm/any_advantages_of_flake8_over_pylint/

Reply:
I switched my linter in Atom to flake8 now and I like it better than pylint. May I remove the pylint stuff? I also like the philosophy of flake8 better.

Reply:
Removed pylint